src/main/java/io/github/bookcrawler/EmpikBookExtractor.java

```diff
@@ -0,0 +1,60 @@
+package io.github.bookcrawler;
+
+import io.github.bookcrawler.entities.BookInfo;
+import org.jsoup.Connection;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class EmpikBookExtractor implements BookExtractor {
+
+    public Iterable<BookInfo> extract() throws IOException {
+        Connection connect = Jsoup.connect("http://www.empik.com/ebooki/promocje/");
+        Set<BookInfo> books = new HashSet<BookInfo>();
```

Comment:
I will check IDE. Previously I had not got that issue.
InteliJ hid it from me, even it was still there.